stack.yaml

```diff
@@ -8,8 +8,93 @@ packages:
   - submit-api
 
 extra-deps:
+  - git: https://github.com/input-output-hk/cardano-crypto
+    commit: 2547ad1e80aeabca2899951601079408becbc92c
+    
+  - git: https://github.com/input-output-hk/cardano-db-sync
+    commit: f37fbea644143b99060a6d068c679ba0be89cf4c
+    subdirs:
+      - cardano-db
+      - cardano-db-sync
+
+  - git: https://github.com/input-output-hk/cardano-shell
+    commit: 601bb4324c258e3c8cbd2d532e5696fd09e2582e
+    subdirs: cardano-shell
+
+  - git: https://github.com/input-output-hk/cardano-prelude
+    commit: f4fa92a94dd974f433f22b401c6fc31a18a50230
+    subdirs:
+    - .
+    - test
+
+  - git: https://github.com/input-output-hk/cardano-sl-x509
+    commit: 43a036c5bbe68ca2e9cbe611eab7982e2348fe49
+    
+  - git: https://github.com/input-output-hk/iohk-monitoring-framework
+    commit: 6faa25842b24ca288974cf6106f23e55c3d4f409
+    subdirs:
+    - contra-tracer
+    - iohk-monitoring
+    - plugins/backend-aggregation
+    - plugins/backend-ekg
+    - plugins/backend-monitoring
+    - plugins/scribe-systemd
+    - tracer-transformers
+
+  - git: https://github.com/input-output-hk/cardano-base
+    commit: 2cc27584bb19bd5be9f1721fd4a2393bb99c6119
+    subdirs:
+    - binary
+    - binary/test
+    - cardano-crypto-class
+    - slotting
+
+  - git: https://github.com/input-output-hk/goblins
+    commit: 26d35ad52fe9ade3391532dbfeb2f416f07650bc
+
+  - git: https://github.com/input-output-hk/cardano-ledger-specs
+    commit: 52afaab4fe99df8fff1e7666e665a923118cfc53
+    subdirs:
+    - byron/semantics/executable-spec
+    - byron/ledger/executable-spec
+    - byron/chain/executable-spec
+    - shelley/chain-and-ledger/dependencies/non-integer
+    - shelley/chain-and-ledger/executable-spec
+    - cardano-ledger
+
+  - git: https://github.com/input-output-hk/cardano-ledger
+    commit: 9ee8a6630a8719ba54554c3ce10c555bf5dff4cc
+    subdirs:
+    - cardano-ledger/test
+    - crypto
+    - crypto/test
+
+  - git: https://github.com/input-output-hk/ouroboros-network
+    commit: ec4f3af58c13a4c20116e9e262b048fb2d27bdc3
+    subdirs:
+    - io-sim
+    - io-sim-classes
+    - network-mux
+    - Win32-network
+    - ouroboros-network
+    - ouroboros-network-framework
+    - ouroboros-network-testing
+    - ouroboros-consensus
+    - ouroboros-consensus/ouroboros-consensus-mock
+    - ouroboros-consensus/ouroboros-consensus-test-infra
+    - ouroboros-consensus-byron
+    - ouroboros-consensus-byronspec
+    - ouroboros-consensus-shelley
+    - ouroboros-consensus-cardano
+    - typed-protocols
+    - typed-protocols-examples
+
+  - git: https://github.com/input-output-hk/cardano-node
+    commit: 3048967df2646cbe7519473600d044eeb2b0dfa8
+    subdirs: cardano-config
+
   - git: https://github.com/input-output-hk/cardano-db-sync
-    commit: 5407a064d4462712a5641295e93cf94d906bd897
+    commit: f37fbea644143b99060a6d068c679ba0be89cf4c
```

Comment:
why so many changes in the `stack` file? Wasn't it upto date with `cabal.project` file?
If so it's worth checking if there is CI check enabled (if it is maybe it is not caputring the

Reply:
The changes in the stack file are incorrect. We referenced the library in the snapshot already.

And no, as far as I know, there's no automated stack / cabal.project conversion going on in CI. Both have to be maintained separately